### ISSUE TITLE
Appendix A: diversity

### ISSUE THREAD

**Issue by yvonneng** [amiaopensource/amia-elections#19]
The bullet about diversity should be expanded to include gender, race, age.

**Comment by michelewinn** [comment]
Agree, this language could even reflect the outcome of the proposed task force activity to "define diversity" for the organization.

**Comment by retokromer** [comment]
Somewhere it should be stated that diversity includes also members that are not based in continental North-America.

**Comment by dericed** [comment]
This bullet could reference the AMIA Diversity Statement (when it's ready to be referenced) rather than additionally define the AMIA meaning of diversity. It might be more consistent in the long run to outsource this definition to an external document.
